marine/marine.py

```diff
@@ -105,6 +114,11 @@ def validate_display_filter(self, display_filter: str) -> bool:
         display_filter = display_filter.encode("utf-8")
         return bool(self._marine.validate_display_filter(display_filter))
 
+    def set_preferences(self, preferences: List[str]) -> int:
+        preferences = [MARINE_PREF_T(*pref.encode("utf-8")) for pref in preferences]
```

Comment:
For a moment there I thought you're pointer-dereferencing in python 😄

Reply:
yes i can see that too 🤣